supplementary_style_guide/glossary_terms_conventions/general_conventions/a.adoc

```diff
@@ -68,6 +68,18 @@ _Specify the system architecture of your cluster, such as `x86_64` or `aarch64`.
 *See also*:
 
 // RHSSO: General; kept as is
+
+[discrete]
```

Comment:
@lpettyjo can you please remove the [discrete] tags in this PR? We no longer use them in our glossary entries (see #365 for details)